src/main/java/com/practicesoftwaretesting/Cart/CartController.java

```diff
@@ -0,0 +1,29 @@
+package com.practicesoftwaretesting.Cart;
```

Comment:
```suggestion
package com.practicesoftwaretesting.cart;
```
Don't use uppercase letters for packages